README.md

```diff
@@ -3,29 +3,105 @@
+
+## Object-Oriented: Streamline Your Code
+
+Realm was built for mobile developers, with simplicity in mind. The idiomatic, object-oriented data model can save you thousands of lines of code.
+
+```swift
+// Define your models like regular Swift classes
+class Dog: Object {
+    @Persisted var name: String
+    @Persisted var age: Int
+}
+class Person: Object {
+    @Persisted(primaryKey: true) var _id: String
+    @Persisted var name: String
+    @Persisted var age: Int
+    // Create relationships by pointing an Object field to another Class
+    @Persisted var dogs: List<Dog>
+}
+// Use them like regular Swift objects
+let dog = Dog()
+dog.name = "Rex"
+dog.age = 1
+print("name of dog: \(dog.name)")
+
+// Get the default Realm
+let realm = try! Realm()
+// Persist your data easily with a write transaction 
+try! realm.write {
+    realm.add(dog)
+}
+```
+## Live Objects: Build Reactive Apps
+Realm’s live objects mean data updated anywhere is automatically updated everywhere. If you need to freeze elements of your data for Reactive frameworks, you can do that, too.
+```swift
+// Open the default realm.
+let realm = try! Realm()
+
+var token: NotificationToken?
+
+let dog = Dog()
+dog.name = "Max"
+
+// Create a dog in the realm.
+try! realm.write {
+    realm.add(dog)
+}
+
+//  Set up the listener & observe object notifications.
+token = dog.observe { change in
+    switch change {
+    case .change(let properties):
+        for property in properties {
+            print("Property '\(property.name)' changed to '\(property.newValue!)'");
+        }
+    case .error(let error):
+        print("An error occurred: (error)")
+    case .deleted:
+        print("The object was deleted.")
+    }
+}
+
+// Update the dog's name to see the effect.
+try! realm.write {
+    dog.name = "Wolfie"
+}
+```
+## Fully Encrypted
+Data can be encrypted in-flight and at-rest, keeping even the most sensitive data secure.
+```swift
+// Generate a random encryption key
+var key = Data(count: 64)
+_ = key.withUnsafeMutableBytes { bytes in
+    SecRandomCopyBytes(kSecRandomDefault, 64, bytes)
+}
+
+// Add the encryption key to the config and open the realm
+let config = Realm.Configuration(encryptionKey: key)
+let realm = try Realm(configuration: config)
+
+// Use the Realm as normal
+let dogs = realm.objects(Dog.self).filter("name contains 'Fido'")
+```
+## Data Sync
+The [MongoDB Realm Sync](https://www.mongodb.com/realm/mobile/sync) service makes it simple to keep data in sync across users, devices, and your backend in real-time.
 
 ## Getting Started
 
-Please see the detailed instructions in our docs to add [Realm Objective-C](https://realm.io/docs/objc/latest/#installation) _or_ [Realm Swift](https://realm.io/docs/swift/latest/#installation) to your Xcode project.
+Please see the detailed instructions in our docs to add [Realm](https://docs.mongodb.com/realm/sdk/ios/install/) to your Xcode project.
```

Comment:
Maybe something like "We support installing Realm via Swift Package Manager, CocoaPods, Carthage, or by importing a dynamic XCFramework. For more information, see the detailed instructions in our docs."